.travis.yml

```diff
@@ -1,11 +1,9 @@
 language: node_js
 node_js:
+  - "0.12"
   - "0.11"
   - "0.10"
   - "0.8"
-matrix:
-  allow_failures:
-    - node_js: "0.11"
```

Comment:
I would prefer to not drop ignoring `0.11` failures. The odd `node` releases are unstable which is why this was introduced.